packages/camera/camera_platform_interface/test/method_channel/method_channel_camera_test.dart

```diff
@@ -70,7 +73,11 @@ void main() {
         // Act
         expect(
           () => camera.createCamera(
-            CameraDescription(name: 'Test'),
+            CameraDescription(
+              name: 'Test',
+              lensDirection: CameraLensDirection.back,
+              sensorOrientation: 0,
```

Comment:
My reasoning was that the `CameraDescription` instance used should always coming from the native side (as a result of the `availableCameras` method). Being such these parameters should always have a value and should never be defaulted to something just because they are not available (this might lead to bugs).

Reply:
Ahhh good point. If users are not going to be creating their own CameraDescription, I'm fine with them having a more involved constructor. Thanks for the explanation!